tests/conftest.py

```diff
@@ -26,6 +26,12 @@ def is_unifi_os_fixture() -> bool:
     return False
 
 
+@pytest.fixture(name="v2")
+def v2_fixture() -> bool:
+    """If use UniFi OS response."""
+    return False
```

Comment:
This shouldn't be needed. You can isolate all logic to within the endpoint_fixture, either you can look at the path to see if it contains v2 or pass in an extra property to mock_get_request which says v1 = True or something

Reply:
I modified this (locally) at the moment in mock_get_request to `data = payload if "v2" in url else {"meta": {"rc": "OK"}, "data": payload}`
Maybe passing in an extra property to mock_get_request, indicating the structure of the data, is a more future-proof way. But I'm missing some inspiration for the naming.

Reply:
Sure, lets go with passing an extra property to mock_get_request where default behaviour goes towards current behaviour.